.github/actions/release/bump-versions/action.yml

```diff
@@ -31,22 +31,6 @@ runs:
         command: install
         args: -f --no-default-features --features "add set-version" --version ^0.8 cargo-edit
 
-    - name: Replace identity_iota version in Wasm bindings
-      shell: bash
-      if: ${{inputs.release-target == 'rust'}}
-      working-directory: bindings/wasm
-      run: |
-        cargo add identity_iota@=${{ inputs.version }} --path=../../identity_iota
-
-    - name: Replace identity versions in stronghold nodejs bindings
-      shell: bash
-      if: ${{inputs.release-target == 'rust'}}
-      working-directory: bindings/stronghold-nodejs
-      run: |
-        cargo add identity_core@=${{ inputs.version }} --path=../../identity_core
-        cargo add identity_iota_core_legacy@=${{ inputs.version }} --path=../../identity_iota_core_legacy
-        cargo add identity_account_storage@=${{ inputs.version }} --path=../../identity_account_storage
-
     - name: Bump Rust crate versions
```

Comment:
We may need an equivalent step to bump the Rust legacy examples?

```
    # cargo workspaces ignores examples_legacy/ but cargo release still tries to version it during publishing.
    - name: Bump Rust examples_legacy version
      shell: bash
      if: ${{inputs.release-target == 'rust'}}
      working-directory: examples_legacy
      run: |
        cargo set-version ${{ inputs.version }}
```

Reply:
Should that have been a problem during the dry-run?
Will add it to be sure, I can't see a downside.

Reply:
Not sure, it could have skipped that step. We found a lot of problems last time during publishing that weren't raised during the `dry-run`.

Reply:
I ran a `dry-run` manual workflow in the fork, where it didn't complain.

Reply:
I added it just to be on the safe side.